menu.js

```diff
@@ -1,12 +1,108 @@
 // eslint-disable-next-line no-unused-vars
+const projects = {
+  pink: {
+    title: 'Pink Way',
+    paragraph: ['A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.'],
+    listed: ['images/divider.png', 'CSS', 'HTML', 'JavaScript'],
+    images: 'images/posts/post1.jpg',
+  },
+  yellow: {
+    title: 'Yellow View',
+    paragraph: ['A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.'],
+    listed: ['images/divider.png', 'Tailwind CSS', 'HTML', 'Ruby'],
+    images: 'images/posts/post2.jpg',
+  },
+  tropical: {
+    title: 'Tropical Cubes',
+    paragraph: ['A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.'],
+    listed: ['images/divider.png', 'React', 'HTML', 'Next.js'],
+    images: 'images/posts/post3.jpg',
+  },
+  windows: {
+    title: 'Windows Chips',
+    paragraph: ['A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.'],
+    listed: ['images/divider.png', 'React', 'Node.js', 'Next.js'],
+    images: 'images/posts/post4.jpg',
+  },
+};
+
+const arr = ['pink', 'yellow', 'tropical', 'windows'];
+for (const index in arr) {
+  const projectName = arr[index];
+  document.getElementById('works').innerHTML += `
+  <section class="${projectName}"">
+  <img class="resize-img" src="${projects[projectName].images}" alt="work-image">
+
+  <div class="post-content">
+    <h2>${projects[projectName].title}</h2>
+    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+    <ul>
+      <li><img src="${projects[projectName].listed[0]}" alt="divider"></li>
+      <li><a href="#">${projects[projectName].listed[1]}</a></li>
+      <li><img src="${projects[projectName].listed[0]}" alt="divider"></li>
+      <li><a href="#">${projects[projectName].listed[2]}</a></li>
+      <li><img src="${projects[projectName].listed[0]}" alt="divider"></li>
+      <li><a href="#">${projects[projectName].listed[3]}</a></li>
+      <li><img src="${projects[projectName].listed[0]}" alt="divider"></li>
+    </ul>
+    <button class="work-button" id="work-button-${projectName}" href="javascript:void(0);" onclick="myModal()">See Project</button>
```

Comment:
- [ ] When a user clicks on the `see project` button it shows all projects. We want to show just the details of the clicked project kindly fix this.

Reply:
@ignatius22 do you have any idea on that? the others ones I've already solved.

Reply:
- You can try giving each of your projects a unique key  so that when you click on the given project it only shows the details of the clicked project